docs/source/references/config.md

```diff
@@ -22,7 +22,7 @@ The following is the default configuration used by Iroha.
   },
   "SUMERAGI": {
     "PEER_ID": {
-      "address": "",
+      "address": "localhost",
```

Comment:
I think there ought to be a discussion about this. 

Localhost bypasses network [card configuration](https://my.oschina.net/fhd/blog/500778). As such it might work in cases where "127.0.0.1" wouldn't. On the surface this is great, but I would hope that the tests that use this configuration would fail if something were wrong.

Reply:
would you like me to use "127.0.0.1"?

Reply:
I would prefer the explicit IPv4 localhost, because it would catch problems with closed ports.